reverse_engineering/api.js

```diff
@@ -182,7 +178,11 @@ module.exports = {
 									bucketInfo,
 									modelDefinitions,
 								}))
-								.sort(data => (app.require('lodash').isEmpty([]) ? -1 : 1));
+								.sort(() => {
+									// originally there was a comparison on: `_.isEmpty(data.entityLevel.inherits) ? -1 : 1`
+									// however, Yugabyte doesn't support PostgreSQL-style table inheritance
+									return -1;
+								});
```

Comment:
If Yugabyte doesn't support table inheritance, I think it's better to delete this logic instead of keeping the reverse of the table list.